python/tank/templatekey.py

```diff
@@ -66,6 +76,27 @@ def __init__(self,
         if not all(self.validate(choice) for choice in self.choices):
             raise TankError(self._last_error)
     
+    @property
+    def choices(self):
+        """
+        :returns:    List of choices available for this key
+        """
+        return self._choices.keys()
+    
+    @property
+    def choice_labels(self):
```

Comment:
I actually like this better ;)  and labelled_choices works fine!  The only change I'd make is just to have labelled_choices return {'ma': 'ma, 'mb': 'mb} when labels aren't specified on the key so the app code doesn't have to handle it.  Will change.